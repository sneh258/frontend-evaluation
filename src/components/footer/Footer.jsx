import React from 'react';
import './footer.css';

export default function Footer() {
  return (
    <div className="footer">
      <div className="footer_content">
        THEMES
        <button className="theme1">1</button>
        <button className="theme2">2</button>
        <button className="theme3">3</button>
      </div>
    </div>
  );
}
