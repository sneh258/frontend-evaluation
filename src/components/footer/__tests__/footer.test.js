import React from 'react';
import Footer from '../Footer';
import { render } from '@testing-library/react';

describe('Footer', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
