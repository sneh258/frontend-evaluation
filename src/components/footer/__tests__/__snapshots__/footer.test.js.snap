// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="footer_content"
    >
      THEMES
      <button
        class="theme1"
      >
        1
      </button>
      <button
        class="theme2"
      >
        2
      </button>
      <button
        class="theme3"
      >
        3
      </button>
    </div>
  </div>
</DocumentFragment>
`;
